tests/operators/test_agnostic_load_file.py

```diff
@@ -477,6 +512,143 @@ def test_aql_gcs_file_to_postgres(self):
         assert df.iloc[0].to_dict()["sell"] == 142.0
 
 
+def upload_test_file_gcp(files, file_conn_id=None):
+    hook = gcs.GCSHook(file_conn_id) if file_conn_id else gcs.GCSHook()
+    for file in files:
+        hook.upload(
+            bucket_name=file["bucket_id"],
+            object_name=file["object_name"],
+            filename=file["filename"],
+        )
+
+
+def upload_test_file_s3(files, file_conn_id=None):
+    hook = s3.S3Hook(aws_conn_id=file_conn_id) if file_conn_id else s3.S3Hook()
+    bucket = hook.get_bucket(files[0]["bucket_id"])
+    for file in files:
+        bucket.upload_file(Key=file["object_name"], Filename=file["filename"])
+
+
+@pytest.mark.parametrize(
+    "file_info",
+    [
+        {
+            "path": "gs://dag-authoring/a",
+            "file_conn_id": "gcp_conn",
+            "load_data": upload_test_file_gcp,
+            "files": [
+                {
+                    "bucket_id": "dag-authoring",
+                    "object_name": "a.csv",
+                    "filename": str(CWD) + "/../data/homes.csv",
+                },
+                {
+                    "bucket_id": "dag-authoring",
+                    "object_name": "aa.csv",
+                    "filename": str(CWD) + "/../data/homes.csv",
+                },
+            ],
+        },
+        {
+            "path": "gs://dag-authoring/a",
+            "file_conn_id": None,
+            "load_data": upload_test_file_gcp,
+            "files": [
+                {
+                    "bucket_id": "dag-authoring",
+                    "object_name": "a.csv",
+                    "filename": str(CWD) + "/../data/homes.csv",
+                },
+                {
+                    "bucket_id": "dag-authoring",
+                    "object_name": "aa.csv",
+                    "filename": str(CWD) + "/../data/homes.csv",
+                },
+            ],
```

Comment:
Please export this to a fixture.

Reply:
We already have a similar fixture:
https://github.com/astro-projects/astro/blob/b1d9b2947be0d594f5c107579a961887f40b6c4f/tests/operators/test_agnostic_load_file.py#L518-L570

Reply:
Used the existing fixture